jscomp/bsb/bsb_build_util.mli

```diff
@@ -37,7 +37,7 @@ val flag_concat : string -> string list -> string
     Then if it is absolute path, it will do thing
     Else if it is relative path, it will be rebased on project's root directory 
 *)
-val convert_and_resolve_path : string -> string
+val convert_and_resolve_path : string -> string -> string
```

Comment:
I checked this function is not used externally, can you comment it to make it not exposed